api/bazel/BUILD

```diff
@@ -0,0 +1 @@
+exports_files(["repository_locations.bzl"])
```

Comment:
I'm exporting this for the `py_binary - //bazel:print_dependencies` rule

Not sure if it's ok to export this - I'm fine to drop the rule, remove this line and use the python script directly.

Reply:
I think this is fine.